features/informatievragen-fields.feature

```diff
@@ -0,0 +1,621 @@
+      | 999993586           |              |                           |                          |           |            |          |            |                                  |                               |                            | 0000 | Onbekend          |                             |                   |              |         |
+
+    @gba
+    Abstract Scenario: met fields gevraagd om <fields>
+      Gegeven het systeem heeft een persoon met de volgende gegevens
+      | naam                | waarde    |
+      | burgerservicenummer | 999993653 |
+      En de persoon heeft de volgende 'verblijfplaats' gegevens
+      | naam                                       | waarde           |
+      | functieAdres.code                          | W                |
+      | functieAdres.omschrijving                  | woonadres        |
+      | datum aanvang adreshouding (10.20)         | 20150808         |
+      | straatnaam (11.10)                         | Boterdiep        |
+      | naam openbare ruimte (11.15)               | Boterdiep        |
+      | huisnummer (11.20)                         | 31               |
+      | postcode (11.60)                           | 3077AW           |
+      | woonplaats (11.70)                         | Rotterdam        |
+      | identificatiecode verblijfplaats (11.80)   | 0599010000208579 |
+      | identificatiecode nummeraanduiding (11.90) | 0599200000219678 |
+      Als personen wordt gezocht met de volgende parameters
+      | naam                | waarde                          |
+      | type                | RaadpleegMetBurgerservicenummer |
+      | burgerservicenummer | 999993653                       |
+      | fields              | <fields>                        |
+      Dan heeft de response een persoon met alleen de volgende 'verblijfplaats' gegevens
+      | naam                             | waarde           |
+      | verblijfadres.straat             | Boterdiep        |
+      | verblijfadres.naamOpenbareRuimte | Boterdiep        |
+      | verblijfadres.huisnummer         | 31               |
+      | verblijfadres.postcode           | 3077AW           |
+      | verblijfadres.woonplaats         | Rotterdam        |
+      | adresseerbaarObjectIdentificatie | 0599010000208579 |
+      | nummeraanduidingIdentificatie    | 0599200000219678 |
+
+      Voorbeelden:
+      | fields                              |
+      | verblijfplaats.verblijfadres.straat |
+      | verblijfadres.straat                |
+      | straat                              |
+      | korteNaam                           |
+      | postcode                            |
+      | nummeraanduidingIdentificatie       |
+      | locatiebeschrijving                 |
+      | woonplaats                          |
+      | land                                |
+      | regel1                              |
```

Comment:
Dus als ik 'regel1' vraag dan krijg ik toch alle in de regels 268 t/m 274 vermelde velden terug?

Reply:
Inderdaad niet zo duidelijk, de eerste 'ik' is de client en de tweede 'ik' de proxy.
Duidelijk dat de client uiteindelijk alleen 'type' en 'regel1' terug krijgt.